lib/plugins/aws/info/getApiKeyValues.js

```diff
@@ -51,9 +51,14 @@ module.exports = {
         })
         .then((apiKeys) => {
           if (apiKeys && apiKeys.length) {
-            info.apiKeys = apiKeys.map((apiKey) =>
-              _.pick(apiKey, ['name', 'value', 'description', 'customerId'])
-            );
+            info.apiKeys = apiKeys.map((apiKey) => {
```

Comment:
I think we don't necessarily need to filter out `undefined` values because looking at how they're used, it does not change the behavior at all and only impacts the implementation and makes it more complicated. 

Instead, let's modify the test to accept the potential `undefined` values for some properties.

Reply:
I agree, @pgrzesik . Code and tests have been updated. Please re-review.